src/applications/disability-benefits/all-claims/tests/utils.unit.spec.jsx

```diff
@@ -542,5 +557,6 @@ describe('isAnswering781aQuestions', () => {
       'view:enterAdditionalSecondaryEvents0': false,
     };
     expect(isAnswering781aQuestions(1)(formData)).to.be.false;
+>>>>>>> e8f9324053a40baab4e31d8df6c81a5d9f6956dc
```

Comment:
merge conflict symbol

Reply:
@rianfowler : Fixed those merge conflicts. Thank you!